extension/packages/nextjs/next.config.js.args.mjs

```diff
@@ -0,0 +1,37 @@
+// This does not work with the current template
+// because we also need to add things one level up of the what
```

Comment:
What do you need outside nextjs?

Reply:
1. Can’t use withSerwist function with the current templating method as it only lets us add things inside the next config. 
    - What withSerwist does: https://github.com/serwist/serwist/blob/main/packages/next/src/index.ts
    - How to use withSerwist for nextjs: https://serwist.pages.dev/docs/next/getting-started
    - Our current template: https://github.com/scaffold-eth/create-eth/commit/e950e6f806fffba571dbf1215ca0d265e84abbcf#diff-f224a81f9a3dbaf31c32522a71b1d6c83a3697cea55ed44920553d6d0653e9a2R5
    - **We need to able to add something in the next config file, but not inside the next config, as shown above in the Serwist link**

Reply:
aja, I understand. I will think about a way to do it...